go/tasks/plugins/k8s/pod/plugin.go

```diff
@@ -19,13 +20,15 @@ import (
 
 const (
 	podTaskType         = "pod"
-	primaryContainerKey = "primary_container_name"
+	PrimaryContainerKey = "primary_container_name"
 )
 
 var (
-	defaultPodBuilder = containerPodBuilder{}
-	podBuilders       = map[string]podBuilder{
-		sidecarTaskType: sidecarPodBuilder{},
+	DefaultPodPlugin = plugin{
```

Comment:
Is there a reason this is a global variable not just initialized in the Array Plugin when it's created?